engine/tests/core38/dispatch/test_dispatch.py

```
@@ -0,0 +1,62 @@
+    on_init,
+)
+from tangl.core38.entity import Entity
+from tangl.core38.registry import Registry
+
+
+class ChildEntity(Entity):
+    """Entity subtype for caller kind filtering."""
+
+
+class TestDispatchFunctions:
+    def test_do_init_chains_ctx_registry_and_inline(self, null_ctx) -> None:
```

Ruff (0.15.0)

[warning] Unused method argument: `null_ctx`

(ARG002)